### DISCUSSION TITLE
My website opens fine on desktop, but shows a blank page on mobile and tablet. What am I doing wrong?

### DISCUSSION THREAD

**Discussion by AnukratiMehta** [community/community#48781] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

I created a portfolio website that works like I want it to on desktop devices but doesn't open at all on mobiles and tablets.

https://anukratimehta.github.io/portfolio-website/

Please help me fix it. TIA!

**Comment by Anmol-Baranwal** [comment, marked as answer]
> First of all kudos for your portfolio.

I was very confused because it worked correctly on chrome developer tools even on resizing the screen.

Since I haven't seen the code so, I checked your portfolio on several devices, and chrome really shows a background color. I would suggest you adjust the code(there may be something in css)

I use chrome on mobile so I checked it for the same.

It works fine in firefox. I am attaching a screenshot for reference.

![Screenshot (3007)](https://user-images.githubusercontent.com/74038190/224123652-ef01ac70-ea3b-47f8-8998-2b06265ae97f.png)

I would suggest you to use this: https://accounts.lambdatest.com/ (used it to check on firefox)

**Reply by Anmol-Baranwal** [reply]
Which platform have you used for hosting? Have you tried vercel?

**Reply by Anmol-Baranwal** [reply]
@AnukratiMehta 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by Anmol-Baranwal** [reply]
@AnukratiMehta 
At least notify us, whether the problem was solved. Since it takes time to write answers that might help the author of the issue.

**Reply by AnukratiMehta** [reply]
Oops, sorry @Anmol-Baranwal, I didn't realize your response wasn't marked as the answer. My bad. Thanks for the help!

**Reply by Anmol-Baranwal** [reply]
Could you please give me more details about the problem, and let me know how it was resolved? 
(Was it CSS code?)
